app/middlewares/users.js

```diff
@@ -11,3 +12,19 @@ exports.emailExists = async (req, res, next) => {
     return next(error);
   }
 };
+
+exports.emailRegistered = async (req, res, next) => {
+  try {
+    const userFound = await userService.emailExists(req.body.email);
+    if (!userFound) throw errors.credentialsError(errorsCatalog.CREDENTIALS_ERROR);
+
+    const passwordMatch = await bcrypt.compare(req.body.password, userFound.dataValues.password);
+    if (!passwordMatch) throw errors.credentialsError(errorsCatalog.CREDENTIALS_ERROR);
+
+    res.locals.user = userFound.dataValues;
```

Comment:
here, must be req.user = userFound.dataValues
Apart from this, you are checking if user exists and
I call the function:
CheckCredentialsAndLoadUser